res/css/_common.scss

```diff
@@ -137,8 +137,14 @@ input[type=search]::-webkit-search-results-decoration {
 // Override Firefox's UA style so we get a consistent look across browsers
 input::placeholder,
 textarea::placeholder {
-    opacity: initial;
+    color: #a7a7a7;
```

Comment:
You shouldn't be setting the color directly here, you should use one of the colors in `_light.scss` and `_dark.scss` (ideally one of the ones that are at the top).